Source/WebCore/css/CSSComputedStyleDeclaration.cpp

```diff
@@ -105,6 +106,17 @@ const FixedVector<CSSPropertyID>& CSSComputedStyleDeclaration::exposedComputedCS
 
 String CSSComputedStyleDeclaration::getPropertyValue(CSSPropertyID propertyID) const
 {
+    auto canUseShorthandSerializerForPropertyValue = [&]() {
+        switch (propertyID) {
+        case CSSPropertyGridTemplate:
```

Comment:
Seems like we can do this for a **lot** more properties, maybe even most, in the longer term. Eventually I’d want this to be reversed in sense, with a list of the shorthand properties that **cannot** use the shorthand serializer. The downside is that `getPropertyCSSValue` for these properties will then be largely untested, since the web platform tests won’t exercise that code much any more.